pkg/cli/admin/release/mirror.go

```diff
@@ -929,3 +928,42 @@ func (o *MirrorOptions) HTTPClient() (*http.Client, error) {
 		Transport: transport,
 	}, nil
 }
+
+func dedupeSortSources(sources []operatorv1alpha1.RepositoryDigestMirrors) []operatorv1alpha1.RepositoryDigestMirrors {
+	unique := make(map[string][]string)
+	var uniqueSources []operatorv1alpha1.RepositoryDigestMirrors
+	var seenSources []string
```

Comment:
You don't need this slice, since you can use the key from `unique` map to do just that.

Reply:
:facepalm: thanks, brain over-working